child/third_party/proto/ibc/core/channel/v1/channel.proto

```diff
@@ -0,0 +1,148 @@
+syntax = "proto3";
```

Comment:
Why are the third_party files under `child/` they should be at root level

Reply:
since main has updated some, and my updates were fairly substantial, I'm going to begin from main again and integrate these changes.